# a list of the detected object information
TrackInfo[]     tracking_objects